connection/Makefile

```diff
@@ -0,0 +1,33 @@
+IMAGE_TAG=latest
```

Comment:
Are we consistently using Makefile or shell script ?

Reply:
Used Makefile to run on *nix and windows (GNUMake). Skill generator also updated to use Makefile https://github.com/CognitiveScale/cortex-cli/pull/304